kvdb-rocksdb/src/lib.rs

```diff
@@ -448,17 +532,31 @@ impl Database {
 			Some(ref cfs) => {
 				let mut batch = WriteBatch::default();
 				let ops = tr.ops;
+
+				self.stats.tally_writes(ops.len() as u64);
+				self.stats.tally_transactions(1);
+
+				let mut stats_total_bytes = 0;
+
 				for op in ops {
 					// remove any buffered operation for this key
 					self.overlay.write()[op.col() as usize].remove(op.key());
 
 					let cf = cfs.cf(op.col() as usize);
 
 					match op {
-						DBOp::Insert { col: _, key, value } => batch.put_cf(cf, &key, &value).map_err(other_io_err)?,
-						DBOp::Delete { col: _, key } => batch.delete_cf(cf, &key).map_err(other_io_err)?,
+						DBOp::Insert { col: _, key, value } => {
+							stats_total_bytes += key.len() + value.len();
+							batch.put_cf(cf, &key, &value).map_err(other_io_err)?
+						}
+						DBOp::Delete { col: _, key } => {
+							// We count deletes as writes.
```

Comment:
I find this a bit confusing, we count deletes as writes, but don't count value size here.
Would be nice to cover it in unit tests at least.